tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -326,6 +326,19 @@ public void setCachingEnabled(boolean enabled) {
                 }
             };
 
+    // map relativeTimePattern counts to numeric examples
```

Comment:
We might need to make this map dependent on locale. That can be done under [CLDR-18025](https://unicode-org.atlassian.net/browse/CLDR-18025) "Further work on examples for relative dates and times".